ts/components/icon/SessionIcon.tsx

```diff
@@ -122,7 +122,7 @@ const animation = (props: {
 };
 
 //tslint:disable no-unnecessary-callback-wrapper
-const Svg = React.memo(styled.svg<StyledSvgProps>`
+const Svg = styled.svg<StyledSvgProps>`
```

Comment:
Just curious but what was `React.memo` breaking on this and the Avatar?

Reply:
no it wasn't. That was part of tracking some performance issues but I don't think it is relevant. Rolling the change back.
I think it makes sense to memoize this component as it is used extensively throughout the app.

Reply:
In that case feel free to keep memo in this case